src/style.css

```diff
@@ -1,3 +1,71 @@
+* {
+  box-sizing: border-box;
+}
```

Comment:
![image](https://github.com/mate-academy/layout_moyo-header/assets/46566190/abdcb376-d408-4502-966f-bc48c315c05e)

Reply:
i don`t know, where i can use box-sizing: border-box. I tried to transfer to html and then to body, but the test did not count, so I decided to put it to *